### PR TITLE
Always show URL bar. closes #1984

### LINKED ISSUE #1984: Add an option to disable "title mode".  Always show URL bar option.

### PR DESCRIPTION
**Pull request by cezaraugusto**
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.

### CODE REVIEW

File: app/extensions/brave/locales/en-US/preferences.properties [hunk 1]
```diff
@@ -77,3 +77,4 @@ alwaysAllow=Always allow
 alwaysDeny=Always deny
 appearanceSettings=Appearance settings
 autoHideMenuBar=Hide the menu bar by default
+disableTitleMode=Disable Title Mode
```
Comment (diracdeltas) [inline app/extensions/brave/locales/en-US/preferences.properties]: I'm not sure people will know what 'Title Mode' is. Maybe change the text to 'Always show URL bar'?

**Comment by diracdeltas** [comment]
otherwise looks good to me. thanks!

File: app/extensions/brave/locales/en-US/preferences.properties (hunk 1 above)
Comment (cezaraugusto) [inline app/extensions/brave/locales/en-US/preferences.properties]: @diracdeltas I agree with you, haven't found a better name for it, so I just copied from the issue.

Should I change only i10n and keep everything else as `disableTitleMode` or change everything to `showUrlBar` instead?

Comment (diracdeltas) [inline app/extensions/brave/locales/en-US/preferences.properties]: fine to just change the l10n string

**Comment by bbondy** [comment]
Looking great!

Please remove the localization files other than English. They are generated only and downloaded from 
Transifex.  Then you can modify the commit with `git rebase -i HEAD~2`, or just squash it all together with `git rebase -i master`.

@bradleyrichter will be excited because he was just asking for this.

You can also set the default to: `process.platform === 'win32'` to avoid us having 3 titles on Windows until we can get rid of the titlebar. It is @bradleyrichter 's preference on Windows.

**Comment by bsclifton** [comment]
Love this change (just pulled and tested it). LGTM (pending @bbondy's feedback)! :smile:

**Comment by cezaraugusto** [comment]
hope it's all good now :dancers:

**Comment by bbondy** [comment]
looks great, thank you!

**Pull request merged**